.rubocop.yml

```diff
@@ -16,6 +16,7 @@ AllCops:
     - 'Rakefile'
     - 'Gemfile.lock'
     - 'config.ru'
+    - 'db/migrate/20220120101655_create_active_storage_tables.active_storage.rb'
```

Comment:
we can just add `'db/**/*'` or `'db/migrate/*'`

Reply:
@PFDeve @skripchenkoVeta I suggest don't any of the approaches.
Instead please user `.rubocop-todo.yml`

Reply:
Lets fix it in the next PR